src/main/java/com/iota/iri/conf/DbConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.iota.iri.conf;
+
+public interface DbConfig extends Config {
+
+    String getDbPath();
+
+    String getDbLogPath();
+
+    int getDbCacheSize();
+
+    String getMainDb();
+
+    boolean isExport();
+
+    boolean isRevalidate();
+
+    boolean isRescanDb();
+
+    interface Descriptions {
+
+        String DB_PATH = "The folder where the DB saves its data.";
+        String DB_LOG_PATH = "The folder where the DB logs info";
+        String DB_CACHE_SIZE = "The size of the DB cache";
```

Comment:
It might be helpful to add that this is measured in KB

Reply:
:+1: